### PR TITLE
Made website more accessible

### PR DESCRIPTION
**Pull request by edi-sipka**
List all points that you have checked, i.e:

- Page titles - No issues found. 

- Image text alternatives** (alt text) - Changed alt text on line 285 to "Html logo" and 293 to "Css logo". 

- Text headings - No issues found. 

- Color contrast - No issues found. 

- Resize - On line 505 we changed max-width to 55%, so about-me paragraph is easier to read. 

- Interaction (labels are optional but recommended as voiceover tools can read them, and should be set as invisible to match the Figma design) - We added on line number 317 property tabindex="1", to make the Skills arrow have Tab interaction.

- Moving content - No issues found. 

- Multimedia - No issues found. 

- The basic structure of the page - No issues found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -312,12 +312,14 @@ <h4 class="caption">Frameworks</h4>
...
           <li class="tabs">
-            <h4 class="caption">Skills</h4>
+            <h4 class="caption">Skills </h4> 
+            <a href="#">
             <img
               class="left-sidebar"
               alt="open-tab"
               src="Images/Disabled.png"
             />
+          </a>
```
Comment (edi-sipka) [inline index.html:322]: We added on line number 316  <a> section .to make the Skills arrow have Tab interaction.

File: style.css [hunk 2]
```diff
@@ -134,7 +134,7 @@ header {
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
-  color: #7f8cff;
```
Comment (edi-sipka) [inline style.css:137]: Instead of capital letters, it is small case now.

File: style.css [hunk 3]
```diff
@@ -511,7 +511,7 @@ header {
     margin-top: -20px;
     background-image: none;
     margin-left: 160px;
-    max-width: 800px;
+    max-width: 55%;
```
Comment (edi-sipka) [inline style.css:514]: On line 514 we changed max-width to 55%, so about-me paragraph is easier to read.

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @edi-sipka      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**